Tella/Scenes/CommonConnectionReport/Views/Draft/DraftView.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  DraftView.swift
+//  Tella
+//
+//  Created by gus valbuena on 6/26/24.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct DraftView<T: ServerProtocol>: View  {
+    @StateObject var viewModel: DraftMainViewModel<T>
+    
+    @State private var menuFrame : CGRect = CGRectZero
+    @State private var shouldShowMenu : Bool = false
+    
+    @EnvironmentObject var mainAppModel: MainAppModel
+    @EnvironmentObject var sheetManager: SheetManager
```

Comment:
it is better to remove and avoid the usage of  `@EnvironmentObject` in our cod. In this situation we can use `viewModel.mainAppModel`

Reply:
got it, I removed it. Just out of curiosity, why is it better to avoid the usage of `@EnvironmentObject`?